contrib/vagrant/start.sh

```diff
@@ -70,7 +70,7 @@ function write_footer() {
     cat <<EOF >> "$filename"
 sleep 2s
 sed -i '/exec/d' /etc/init/cilium-net-daemon.conf
-echo 'exec cilium -D daemon run -n ${ipv6_addr} ${ipv4_options}-t vxlan -c "${NODE_IP_BASE}${FIRST_IP_SUFFIX}:8500"' >> /etc/init/cilium-net-daemon.conf
+echo 'exec cilium -D daemon run -n ${ipv6_addr} ${ipv4_options}-t vxlan --ipv4 -c "${NODE_IP_BASE}${FIRST_IP_SUFFIX}:8500"' >> /etc/init/cilium-net-daemon.conf
```

Comment:
Why not setting `IPV4=1` at the top of the file? That option is being verified at line 66-68